cg/services/order_validation_service/models/sample.py

```diff
@@ -8,6 +8,7 @@ class Sample(BaseModel):
     comment: str | None = None
     container: ContainerEnum
     container_name: str | None = None
+    generated_lims_id: str | None = None  # Will be populated by LIMS and stored as the internal id
```

Comment:
Technically a tech-savvy user with curl knowledge can set their own internal ids with this change. Is that something we want to fix now or is that for the future?

Reply:
fixed with the private attribute